pkg/hubble/relay/pool/manager.go

```diff
@@ -322,8 +322,9 @@ func (m *PeerManager) connect(p *peer, ignoreBackoff bool) {
 	}
 
 	m.opts.log.WithFields(logrus.Fields{
-		"address": p.Address,
-	}).Debugf("Connecting peer %s...", p.Name)
```

Comment:
these logs should probably be structured not interpolated

Reply:
I'm not sure I understand what you mean, could you elaborate?

Reply:
structured logging (which logrus is good at) essentially means keeping the error message consistent while providing additional metadata that can be searched/index.

for example if you failed to open the file foo, a structured log would be something like `error=failed to open file; filename=foo` rather than `failed to open filename foo`

Reply:
@glibsm thanks for the clarification, I've added a commit to the PR so that relay uses more structured logging, PTAL.